messages/edge_applications/messages.go

```diff
@@ -32,13 +32,12 @@ var (
 	--type string       The type of Edge application
 	cdn                 Create an edge application to cache and deliver your content.
 	javascript          Create a serverless Javascript application on edge.
-	flareact            Create a serverless Flareact application on edge.
 	nextjs              Create a serverless NextJS edge-runtime application on edge.`
 	EdgeApplicationsInitShortDescription  = "Initializes an Edge Application"
 	EdgeApplicationsInitLongDescription   = "Defines primary parameters based on a given name and application type to start a Edge Application on Azion’s platform"
 	EdgeApplicationsInitRunningCmd        = "Running init step command:\n\n"
 	EdgeApplicationsInitFlagName          = "The Edge application's name"
-	EdgeApplicationsInitFlagType          = "The type of Edge application <javascript | flareact | nextjs>"
+	EdgeApplicationsInitFlagType          = "The type of Edge application <javascript | nextjs>"
```

Comment:
just a reminder that we will have to change this once static type is implemented